src/Mvc/Mvc.ViewFeatures/src/AntiforgeryExtensions.cs

```diff
@@ -10,6 +10,9 @@
 
 namespace Microsoft.AspNetCore.Mvc.ViewFeatures
 {
+    /// <summary>
+    /// Static class that adds extension methods to <see cref="IAntiforgery"/>.
```

Comment:
```suggestion
    /// Static class that adds extension methods to <see cref="IAntiforgery"/>. This class cannot be inherited.
```

Per the style recommendations [here](https://github.com/dotnet/dotnet-api-docs/wiki/Summary%3A-Class).